src/test/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthServiceTest.java

```diff
@@ -171,8 +171,7 @@ void GIVEN_bad_group_configuration_WHEN_start_service_THEN_service_in_error_stat
         ignoreExceptionOfType(context, IllegalArgumentException.class);
         ignoreExceptionOfType(context, UnrecognizedPropertyException.class);
 
-        startNucleusWithConfig("badGroupConfig.yaml", State.ERRORED);
-
+        startNucleusWithConfig("badGroupConfig.yaml", State.RUNNING);
```

Comment:
so we're changing the behavior now that it will be running rather than errored? Is that desired?

Reply:
I think we should be erroring out to cause a failed and rolled back deployment rather than silently failing.

Reply:
will adjust